src/main/resources/templates/pieGraph.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="~{fragments :: header}">
+    <title>PIE</title>
+
+
+</head>
+<body>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-3d.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+
+<h1> Graphs and Data </h1>
+
+<nav th:replace="~{fragments :: nav}"></nav>
+
+<div id="container" style="height: 400px"></div>
+<script th:inline="javascript">
+		Highcharts.chart('container', {
+			chart : {
+				type : 'pie',
+				options3d : {
+					enabled : true,
+					alpha : 50,
+					beta : 0
+				}
+			},
+			title : {
+				text : 'Tone Breakdown'
+			},
+			tooltip : {
+				pointFormat : '{series.name}: <b>{point.percentage:.1f}%</b>'
+			},
+			plotOptions : {
+				pie : {
+					allowPointSelect : true,
+					cursor : 'pointer',
+					depth : 35,
+					dataLabels : {
+						enabled : true,
+						format : '{point.name}'
+					}
+				}
+			},
+			series : [ {
+				type : 'pie',
+				name : 'Ratio',
+				data : [
+				[ 'Anger', [[${anger}]] ],
+				[ 'Fear', [[${fear}]] ],
+				[ 'Joy', [[${joy}]] ],
+				[ 'Sadness', [[${sadness}]] ],
+				[ 'Analytical', [[${analytical}]] ],
+				[ 'Confident', [[${confident}]] ],
+				[ 'Tentative', [[${tentative}]] ]
+				]
+			} ]
+		});
+
+
+
+
+
+
+
+
+
+
+
+
+

```

Comment:
So many blank lines.  Otherwise great formatting, and good job getting this working.